pyiron/vasp/base.py

```diff
@@ -1619,12 +1619,17 @@ def set_rwigs(self, rwigs_dict):
 
         Args:
             rwigs_dict (dict): Dictionary of species and corresponding radii.
+                (structure has to be defined before)
         """
+        if not isinstance(rwigs_dict, dict):
+            raise AssertionError("'rwigs_dict' has to be a dict!")
+        if not all([isinstance(val, (int, float)) for val in rwigs_dict.values()]):
+            raise ValueError("The values of 'rwigs_dict' has to be floats!")
         species_keys = self.structure.get_number_species_atoms().keys()
         rwigs_keys = rwigs_dict.keys()
         for i in species_keys:
```

Comment:
Sorry for the annoying nitpick! But `i` usually indices a number. In this case it's better to write something like ` for sp in species_keys:`. This is an optional change!

Reply:
or just k instead of i ;-)

Reply:
yes, you're totally right!